Gemfile

```diff
@@ -8,10 +8,9 @@ source "https://rubygems.org"
 # Bundler should refer to the gemspec for any dependencies.
 gemspec
 
-if Gem.ruby_version.to_s.start_with?("2.5")
-  # 16.7.23 required ruby 2.6+
-  gem "chef-utils", "< 16.7.23" # TODO: remove when we drop ruby 2.5
-end
+gem "chef-winrm", git: "https://github.com/chef/chef-winrm.git", branch: "jfm/chef-winrm-update"
+gem "chef-winrm-fs", git: "https://github.com/chef/chef-winrm-fs.git", branch: "jfm/winrm-fs-update-2"
+gem "chef-winrm-elevated", git: "https://github.com/chef/chef-winrm-elevated.git", branch: "jfm/chef-winrm-elevated-update"
```

Comment:
Are these temporary changes until upstream libraries fix it? Or are we planning to publish these gems in future to rubygems ?